### PR TITLE
VU-meter library design 2

### PR DESCRIPTION
**Pull request by TenghuiZhang**
#337 was closed accidentally, and this PR continues the work from #337. 

It includes:
- The new VU-Meter class
- Revising comments related to Worklet and SPN recording demo.

### CODE REVIEW

**Comment by hoch** [comment]
Please expand the PR description to say what this PR is for. I am not sure what I need to focus. A good PR description usually has bullet points of goals/key results.

Providing the right context can help to avoid unnecessary communication.

File: src/library/FIFO.js [hunk 1]
```diff
@@ -1,12 +1,10 @@
 /**
- * A First-In-First-Out (FIFO) class that maintains a queue of
- * elements.  New elements are added to the tail, and the oldest
- * element is removed from the head when the FIFO is full.  If the
- * size of the queue is 1, it will always contain the newest element
- * that was inserted.
+ * A First-In-First-Out (FIFO) class that only design for vu-meter to
+ * maintains a queue of elements. In class does not has pull,
+ * getBufferLength function. It includes the getMinValue function for
+ * tracking the minimum value in the FIFO class.
```
Comment (hoch) [inline src/library/FIFO.js:5]: Perhaps:

`@classdesc A FIFO (First-In-First-Out) class that is specifically designed for the VUMeter class in the library directory. This class does not have a method to get or drop a head value, but it includes one for getting a minimum value form the queue.`

https://jsdoc.app/tags-classdesc.html: this tag seems better for this purpose.

**Comment by TenghuiZhang** [comment]
Waveform screencasting:
[Waveform.webm](https://github.com/GoogleChromeLabs/web-audio-samples/assets/32180291/8b930e0a-b670-46d2-a025-ec8ad84ad027)

File: src/library/VUMeter.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ import FIFO from './FIFO.js';
  * considered silent. When the VUMeter is created, it requires an
  * AnalyserNode for real-time audio input data. This class requires a
  * canvas element and FIFO class. The FIFO class is used to determine
- * how fast the VUMeter gets updated. The Waveform bar size is
+ * how fast the VUMeter gets updated. The height of vu-meter bar is
```
Comment (hoch) [inline src/library/VUMeter.js:13]: "The height of the VUMeter" seems better.

File: src/library/VUMeter.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ import FIFO from './FIFO.js';
...
  * determined by the minimum number in the FIFO class.
  * @class
```
Comment (hoch) [inline src/library/VUMeter.js:15]: Use `@classdesc` just like above.

**Review by hoch — APPROVED** [review]
LGTM with nits

**Pull request merged**